### DISCUSSION TITLE
gatsby develop command throws syntax error

### DISCUSSION THREAD

**Discussion by sleep-sort** [gatsbyjs/gatsby#33938] [Help]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report directly related to Gatsby. Please post those things in GitHub Discussions: https://github.com/gatsbyjs/gatsby/discussions

### Description

I am not sure if this is a real issue considering I can't find anything in the open issues for this, but fwiw running gatsby develop throws this syntax error: 
```
.nvm/versions/node/v10.24.1/lib/node_modules/gatsby-cli/node_modules/gatsby-telemetry/lib/telemetry.js:39
  store = new _eventStorage.EventStorage();
```

stack: 
```
SyntaxError: Unexpected token =
    at Module._compile (internal/modules/cjs/loader.js:723:23)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
```

feel free to close if this is the wrong place for this - I'm sorry if this is totally a bother! 

### Reproduction Link

n/a

### Steps to Reproduce

1. fresh install of gatsby cli 
2. open an existing gatsby project (in this instance I pulled down the gatsby starter blog) 
3. run gatsby develop (or really any gatsby command) and observe error
...

### Expected Result

the command would ideally run as expected 

### Actual Result

error :/ 

### Environment

```shell
I can't even run this without getting the error thrown
```

**Comment by LekoArts** [comment]
Hi!

With what you provided I'd guess you're running an ancient version of Node:

```
.nvm/versions/node/v10.24.1
```

You'll need to use at least Node 14.15.0 with Gatsby

**Reply by daltongarbin** [reply]
It worked for me. Thanks!

**Comment by colemerrick** [comment]
Worth noting if you absolutely need to run an old version of gatsby, you can always downgrade the gatsby-cli and whatever else you need temporarily
